tool/tctl/common/collection.go

```diff
@@ -918,6 +918,37 @@ func (c *kubeServerCollection) writeJSON(w io.Writer) error {
 	return trace.Wrap(err)
 }
 
+type kubeClusterCollection struct {
+	clusters []types.KubeCluster
+	verbose  bool
+}
+
+func (c *kubeClusterCollection) resources() (r []types.Resource) {
+	for _, resource := range c.clusters {
+		r = append(r, resource)
+	}
+	return r
+}
+
+func (c *kubeClusterCollection) writeText(w io.Writer) error {
```

Comment:
nit: Can you add a comment with some example output? It makes it easier to reason about the output when you can see what it should look like.